test/modules/TestResultCoordinator/Reports/TestSummary.cs

```diff
@@ -1,24 +1,21 @@
 // Copyright (c) Microsoft. All rights reserved.
 namespace TestResultCoordinator.Reports
 {
-    using System;
+    using System.Collections.Generic;
     using System.Linq;
     using Microsoft.Azure.Devices.Edge.Util;
 
     class TestSummary
     {
-        internal TestSummary(string testBuildNumber, string deviceId, ITestResultReport[] testResultReports, string blobContainerUri)
+        internal TestSummary(Dictionary<string, string> testInfo, ITestResultReport[] testResultReports, string blobContainerUri)
         {
-            this.TestBuildNumber = Preconditions.CheckNonWhiteSpace(testBuildNumber, nameof(testBuildNumber));
-            this.DeviceId = Preconditions.CheckNonWhiteSpace(deviceId, nameof(deviceId));
+            this.TestInfo = Preconditions.CheckNotNull(testInfo, nameof(testInfo));
             this.TestResultReports = Preconditions.CheckNotNull(testResultReports, nameof(testResultReports));
             this.IsPassed = testResultReports.All(r => r.IsPassed);
             this.BlobContainerUri = blobContainerUri ?? string.Empty;
         }
 
-        public string TestBuildNumber { get; }
-
-        public string DeviceId { get; }
+        public Dictionary<string, string> TestInfo { get; }
```

Comment:
Can you verify if SortedDictionary will output in alphabetic order when serialization? if so, please use it.

Reply:
Tested with dotnet fiddle. It does so I made the change.